code/object/object.h

```diff
@@ -304,9 +304,34 @@ void obj_client_post_interpolate();
 // move an observer object in multiplayer
 void obj_observer_move(float frame_time);
 
-// Goober5000
+/**
+ * @brief Checks if the given object is docked with anyone.
+ *
+ * @returns Nonzero if docked, or
+ * @returns 0 if not docked
+ *
+ * @author Goober5000
+ */
 int object_is_docked(object *objp);
+
+/**
+* @brief Checks if the given object is dead-docked with anyone.
```

Comment:
Added "@details" section on there. It's when the object is dead, and still has objects docked to it. Presumably this is to allow a dying parent object do deal damage only to its direct children instead of the entire dock chain (children of children).